eslint.config.mjs

```diff
@@ -0,0 +1,21 @@
+// @ts-check
+
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default tseslint.config(
+    eslint.configs.recommended,
+    ...tseslint.configs.recommended,
+    {
+        rules: {
+            // TODO: Remove ignored rules and fix the code
+            "@typescript-eslint/no-explicit-any": "off",
+            "@typescript-eslint/no-this-alias": "off",
+            "@typescript-eslint/no-unused-vars": "off",
+            "@typescript-eslint/no-var-requires": "off",
+            "no-undef": "off",
+            "no-unreachable": "off",
+            "no-useless-catch": "off",
+        }
```

Comment:
These are rules that the original `tslint` ignored and/or were not covered by the defaults. They have been added, so the code passes lint checks "as is".

For example, here I set: `"@typescript-eslint/no-explicit-any": "off"` and the original `tslint.json` had: `"no-unsafe-any": false,` 

My plan is to enable more severe rules as we move forward. The current configuration works as a baseline.

Reply:
Will this break the customers code?

Reply:
Nope, on the contrary, these rules only affect the code in the plugin, which remains the same at the moment.